celerybeat/Dockerfile

```diff
@@ -8,14 +8,16 @@ ENV DEBIAN_FRONTEND="noninteractive"
 
 RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/stretch-backports.list
 
-# Pillow/Imaging: https://pillow.readthedocs.io/en/latest/installation.html#external-libraries
 RUN apt-get update -qy && apt-get dist-upgrade -qy && apt-get install -o Dpkg::Options::='--force-confnew' -qy \
     git curl \
-    libz-dev libfreetype6-dev \
     libmemcached-dev \
+    # Pillow/Imaging: https://pillow.readthedocs.io/en/latest/installation.html#external-libraries
+    libz-dev libfreetype6-dev \
     libtiff-dev libjpeg-dev libopenjp2-7-dev libwebp-dev zlib1g-dev \
-    graphviz \
-    locales && apt-get -qy autoremove && apt-get -qy autoclean
+    # Postgres client library to build psycopg2
+    libpq-dev \
+    locales \
+    gcc && apt-get -qy autoremove && apt-get -qy autoclean
```

Comment:
I don't know that I want to do this immediately but it feels like we should at some point switch to the builder pattern so we can ship only the runtime dependencies